py/torch_tensorrt/dynamo/conversion/impl/unary/base.py

```diff
@@ -9,6 +12,23 @@
 )
 from torch_tensorrt.dynamo._SourceIR import SourceIR
 from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor, TRTUnaryOp
+from typing import Union, Callable, Any, Optional
+
+SUPPORTED_ATEN_TORCH_UNARY_OPS: Dict[
+    TRTUnaryOp, Union[Callable[[Any], Any], Callable[[Any, Any], Any]]
+] = {}
```

Comment:
The support dictionary could be pre-populated here directly, since the `get_python_op_from_trt_unary_op` function below populates it in a static/fixed way regardless.

Reply:
Additionally, consider renaming to something like `SUPPORTED_TRT_UNARY_PYTHON_OP_CONVERSIONS`, to reflect that the dictionary maps to Python operators.